content/docs/working-groups/_index.md

```diff
@@ -39,6 +39,8 @@ New working groups should add a subdirectory and can include the contents of the
 adding their own details. Each new working group should also added to the list in `README` at the
 root of this repository.
 
+<a href="/docs/working-groups/template"> Go to template </a>
```

Comment:
Can we move the template into a code block on the "setting things up" section of the "Form a new working group" procedure rather than as a sibling to the other working groups?

Reply:
We've added the Template link index to "Form a new working group" procedure and eliminated it from the list of working groups so is no longer a sibling to the other WG. Would you like us to delete the link in the index working group page as well?

Reply:
Yeah, I think this section ("Template") of the working groups index page can be removed - it seems to fit more with the "forming a working group" procedure than the general "running a working group" advice that the working group index page has.